### PR TITLE
[ST] Add Strimzi 0.45.0 to the upgrade/downgrade tests

### PR DESCRIPTION
**Pull request by im-konge**
### Type of change

- Update

### Description

This PR updates the upgrade/downgrade tests with Strimzi 0.45.0 after the release.

### Checklist

- [x] Make sure all tests pass

### CODE REVIEW

File: systemtest/src/test/resources/upgrade/BundleDowngrade.yaml [hunk 1]
```diff
@@ -30,18 +30,18 @@
...
+  toUrl: https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.45.0/strimzi-0.45.0.zip
   fromKafkaVersionsUrl: HEAD
   additionalTopics: 2
   imagesAfterOperations:
-    zookeeper: strimzi/kafka:0.44.0-kafka-3.8.0
-    kafka: strimzi/kafka:0.44.0-kafka-3.8.0
-    topicOperator: strimzi/operator:0.44.0
-    userOperator: strimzi/operator:0.44.0
+    zookeeper: strimzi/kafka:0.45.0-kafka-3.8.0
+    kafka: strimzi/kafka:0.45.0-kafka-3.8.0
```
Comment (ppatierno) [inline systemtest/src/test/resources/upgrade/BundleDowngrade.yaml:42]: we have 3.9.0 now, should this be updated? or it's about upgrading strimzi but taking the same Kafka version?

**Review by ppatierno — APPROVED** [review]
LGTM but I left one question.

File: systemtest/src/test/resources/upgrade/BundleDowngrade.yaml (hunk 1 above)
Comment (im-konge) [reply systemtest/src/test/resources/upgrade/BundleDowngrade.yaml:42]: it's about changing the Strimzi version, the Kafka version is changed when new Kafka version is added.

**Review by see-quick — APPROVED** [review]

**Pull request merged**